### DISCUSSION TITLE
Issue with Hard drive storage full(apps would refresh) so tried to free storage but its gone badly wrong

### DISCUSSION THREAD

**Discussion by Mopsoc** [IceWhaleTech/CasaOS#1610] [General]
Hi

Had an issue about apps wouldn't refresh and plex wouldn't work seemed to be caused by the hard drive storage on the Zimaboard was full. I then went to free up storage link copied the first link and pasted it into the my terminal log but it didn't complete the process and I couldn't get past the Y/n question could type Y as it looked as the terminal log frozen so I tried to the repeat the process but could never get as far. Then when I logged back into my server all I have is the casa os  page with no widgets, no apps and can't see the terminal log and settings in the top left conner has disappeared aswell as everything else. Any help how to fix this would be appreciated.

T

**Comment by dragonfire1119** [comment]
Can you run the CasaOS Healthcheck big-bear-script and paste the output? https://youtu.be/mTqtkmjyGeQ

**Reply by Mopsoc** [reply]
Unfortunately I have no access to the terminal log it has disappeared with everything else all I have is the casa os screen with the 4 items in the bottom right conner(git hub icon, share casa os icon etc)

**Reply by dragonfire1119** [reply]
Can you SSH into the server running CasaOS?

**Reply by Mopsoc** [reply]
Could you go step by step in how i'd go about doing this? Listened to a few youtube videos there it didn't help much

**Reply by Mopsoc** [reply]
Only had Plex on the Zimaboard, Fixed the storage issue by reinstalling casa os, then turn off enable thumbnail on plex and my hard drive storage is now fine and server is working good, Thanks for all the help
